sdcm/report_templates/results_longevity.html

```diff
@@ -8,6 +8,14 @@ <h3>
             <li>Scylla version: {{ scylla_version }} ({{ scylla_ami_id }})</li>
             <li>Instance type: {{ scylla_instance_type }}</li>
             <li>Number of scylladb nodes: {{ number_of_db_nodes }}</li>
+            {% if nodes_shards %}
+            <ul>
```

Comment:
move `<ul>` inside the `<li>Scylla running with...`

Reply:
changed to table